### PR TITLE
[CELEBORN-618] [FLINK] worker side adds partition split configuration options

### PR DESCRIPTION
**Pull request by zhongqiangczq**
### What changes were proposed in this pull request?
now flink client hasn't support partition split, so server side add a configuration to close split.

### CODE REVIEW

**Review by FMX — APPROVED** [review]
LGTM. There are conflicts to be solved first.

File: worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala [hunk 1]
```diff
@@ -76,6 +77,7 @@ class PushDataHandler extends BaseMessageHandler with Logging {
     storageManager = worker.storageManager
     shutdown = worker.shutdown
     conf = worker.conf
+    workerPartitionSplitEnabled = conf.workerPushHeartbeatEnabled
```
Comment (waitinfuture) [inline worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala:80]: typo? should be ```workerPartitionSplitEnabled```

Comment (waitinfuture) [inline worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala:80]: typo? should be workerPartitionSplitEnabled

**Review by waitinfuture — CHANGES_REQUESTED** [review]

File: worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala (hunk 1 above)
Comment (zhongqiangczq) [reply worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala]: yes. it's my fault. it has been corrected.

**Review by waitinfuture — APPROVED** [review]
LGTM, thanks!

**Pull request merged**